documentation/Color system.md

```diff
@@ -75,6 +75,7 @@ The interactive role is used to express interactivity in components. It is used
 | <pre>--p-interactive-selected</pre>         | For use as a surface color in selected interactive elements, in components such as option list and resource list. `-inverse`, `-light`, and `-dark` variants available.                  | ![](https://www.gifpng.com/64x64/f1f4fe/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#f1f4fe | ![](https://www.gifpng.com/64x64/020e22/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#020e22 |
 | <pre>--p-interactive-selected-hovered</pre> | For use as a surface color in selected interactive elements that are hovered, in components such as option list and resource list. `-inverse`, `-light`, and `-dark` variants available. | ![](https://www.gifpng.com/64x64/d4defc/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#d4defc | ![](https://www.gifpng.com/64x64/071c3b/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#071c3b |
 | <pre>--p-interactive-selected-pressed</pre> | For use as a surface color in selected interactive elements that are pressed, in components such as option list and resource list. `-inverse`, `-light`, and `-dark` variants available. | ![](https://www.gifpng.com/64x64/b9c9f9/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#b9c9f9 | ![](https://www.gifpng.com/64x64/0d2a54/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#0d2a54 |
+| <pre>--p-icon-on-interactive</pre>          | For use as a fill color for icons on critical actions.                                                                                                                                   | ![](https://www.gifpng.com/64x64/ffffff/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#ffffff | ![](https://www.gifpng.com/64x64/ffffff/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#ffffff |
```

Comment:
I think we can replace `critical actions` with `interactive elements`

Reply:
> > @jessebc are the 2 different styles for mobile and desktop intentional?
> 
> Yes and no. We were prototyping out 2 different directions in browser to see which direction we felt more confident in. I'll update the figma file to align with our findings.

The version you are seeing in this PR makes it so that both checkbox and radio share the same CSS if that influences the decision. It probably shouldn't but if it comes to a coin toss.